prisma/schema.prisma

```diff
@@ -17,10 +17,11 @@ datasource db {
 
 model User {
   id       Int         @id @default(autoincrement())
-  name     String
+  name     String?
   email    String      @unique
-  username String      @unique
-  roles    Roles[]
+  username String?     @unique
```

Comment:
same here, lets user the email as the default username for now, while we don't have a step to select the username.

When this becomes something editable, the user will be able to change it